docs/docusaurus/docs/core/introduction/community_resources.md

```diff
@@ -1,8 +1,22 @@
 ---
-title: Contribute
-id: contributing
+title: 'Community Resources'
 ---
+import GxCloudAdvert from '/static/docs/_static_components/_gx_cloud_advert.md'
 
+Great Expectations (GX) is committed to supporting and growing the GX community. It’s not enough to build a great tool. GX wants to build a great community as well.
+
+Open source doesn’t always have the best reputation for being friendly and welcoming, and that makes us sad. Everyone belongs in open source, and GX is dedicated to making you feel welcome.
+
+## Contact Great Expectations
+Join the GX [public Slack channel](https://greatexpectations.io/slack). Before you post for the first time, review the [Slack Guidelines](https://discourse.greatexpectations.io/t/slack-guidelines/1195).
```

Comment:
I would rename this section to "Connect with our community" and add some text describing the opportunity to connect with other users of GX and see how they're using it. I would also put this below the "Ask a question" section.

Reply:
Agreed - I'd even suggest moving"Connect with our community" below "Contribute code or documentation."

Reply:
I'll bump it below "Contribute code or documentation" -- it seems like an interruption if it's put between "Ask a question" and "File a bug or feature request."